src/capitalsFirst.js

```diff
@@ -21,7 +21,12 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  const wordsArray = str.split(' ');
+  let upperWords = wordsArray.filter(word => word[0] >= 'A' && word[0] <= 'Z');
```

Comment:
do not hardcode strings inside code move it to constants outside of function and name with UPPER_CASE
ex.: const FIRST_LOWERCASE_LETTER = 'a';
Also, you can move `word[0] >= 'A' && word[0] <= 'Z'` to isUppercaseLetter helper function